src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -418,17 +418,15 @@ function get_item_html($itemkey) {
 $widgetperiod = isset($config['widgets']['period']) ? $config['widgets']['period'] * 1000 : 10000;
 $widgetperiod += 1000;
 
-$filesystems = get_mounted_filesystems();
-
-if (strlen($user_settings['widgets']['system_information']['filter']) > 0) {
-	$itemsshown = array_diff($validNames, explode(",", $user_settings['widgets']['system_information']['filter']));
+if (strlen($user_settings['widgets'][$widgetkey]['filter']) > 0) {
```

Comment:
Needs to use $widgetkey which has the specific key for each copy of the widget 'system_information-0' 'system_information-1'